autora/skl/bms.py

```diff
@@ -63,10 +64,11 @@ def __init__(
             ts: contains a list of the temperatures that the parallel ms works at
         """
         self.ts = ts
-        self.prior_par = prior_par
+        self.prior_par = get_priors()[0]
```

Comment:
with this change, i'm a little confused as to why `prior_par` is now grayed out within the __init__ function above (at least it is for when i open this file in PyCharm)

Reply:
good spot! I shouldn't have made this change. It would prevent us from inputting our own priors. All change this back and be consistent by calling ops the same way

Reply:
I remember why I did this now. The problem is that I have to call in ops to add the custom op to it. However, because priors and ops would then be coming in separately, it would be possible to get a mismatch. I am realising that it would be possible to get a mismatch already since currently ops are brought in later. I believe the only way to fix this is for bms to take in both the priors and ops at the same time. Since this would probably create a breaking change somewhere in our example documentation, I'll add an issue to address this later.

Reply:
I will undo what I did here though so it doesn't grey out for you.

Reply:
ah, i see. do you think this is something we should address sooner rather than later?